python/python_api/cloe/__init__.py

```diff
@@ -1,2 +1,2 @@
 from ._runner import TestRunner, Simulation
-from ._cloe_bindings import CallbackResult
+from _cloe_bindings import CallbackResult
```

Comment:
Is this necessary / safe?

Reply:
Above you're still importing from `._runner`, implying that mechanism still works.
So what is removing relative import gaining here?

Reply:
In editable mode the _cloe_bindings.so file is not next to the python file so relative import won't work.
Python finds it through the PYTHONPATH env variable.